src/renderers/shaders/ShaderChunk/common.glsl

```diff
@@ -9,6 +9,8 @@
 #define whiteCompliment(a) ( 1.0 - saturate( a ) )
 
 float square( const in float x ) { return x*x; }
+float cube( const in float x ) { return x*x*x; }
```

Comment:
Doesn't need to be done for this PR, but maybe we should rename these to `pow2`, `pow3` and `pow4`.